homeassistant/components/overkiz/climate_entities/__init__.py

```diff
@@ -18,7 +16,9 @@
     UIWidget.ATLANTIC_ELECTRICAL_HEATER_WITH_ADJUSTABLE_TEMPERATURE_SETPOINT: AtlanticElectricalHeaterWithAdjustableTemperatureSetpoint,
     UIWidget.ATLANTIC_ELECTRICAL_TOWEL_DRYER: AtlanticElectricalTowelDryer,
     UIWidget.ATLANTIC_HEAT_RECOVERY_VENTILATION: AtlanticHeatRecoveryVentilation,
-    UIWidget.ATLANTIC_PASS_APC_HEATING_AND_COOLING_ZONE: AtlanticPassAPCHeatingAndCoolingZone,
+    # ATLANTIC_PASS_APC_HEATING_AND_COOLING_ZONE works exactlty the same as ATLANTIC_PASS_APC_HEATING_ZONE
+    UIWidget.ATLANTIC_PASS_APC_HEATING_AND_COOLING_ZONE: AtlanticPassAPCHeatingZone,
```

Comment:
Do you own the `AtlanticPassAPCHeatingAndCoolingZone`? If I recall, this device can also be set to a cooling mode and it seems that `io:PassAPCOperatingModeState` does support this?

For now, let's merge this change, but it would be good to follow-up and see if we can extend the `AtlanticPassAPCHeatingAndCoolingZone` to support cooling as well. (if an user can verify that the official app supports this).

Reply:
Yes I do have this device.  
In setup, it does support cooling.  
But in reality, there is no cooling feature (at least for my device)

AFAIK it was never implemented in Ha-Tahoma, let's wait for somebody to ask support for it (if any...)

Reply:
@nyroDev does it support cooling in your vendor app (TaHoma / CozyTouch)? If not, let's not implement this. Sometimes the setup definition offers a lot more than is actually implemented and working.

Reply:
No
![Capture d’écran de 2022-11-30 00-30-54](https://user-images.githubusercontent.com/265662/204671321-c13bb770-278a-4830-958b-0f115803df20.png)

Reply:
Let's resolve this comment for now, we can always handle on feedback.